x/ibc-transfer/client/cli/tx.go

```diff
@@ -40,24 +37,27 @@ func GetTransferTxCmd(cdc *codec.Codec) *cobra.Command {
 			sender := cliCtx.GetFromAddress()
 			srcPort := args[0]
 			srcChannel := args[1]
-			destHeight, err := strconv.Atoi(args[2])
-			if err != nil {
-				return err
-			}
+			receiver := args[2]
 
-			// parse coin trying to be sent
-			coins, err := sdk.ParseCoins(args[4])
+			coins, err := sdk.ParseCoins(args[3])
 			if err != nil {
 				return err
 			}
 
-			msg := types.NewMsgTransfer(srcPort, srcChannel, uint64(destHeight), coins, sender, args[3])
+			timeoutHeight := viper.GetUint64(flagTimeoutHeight)
+			timeoutTimestamp := viper.GetUint64(flagTimeoutHeight)
+
+			msg := types.NewMsgTransfer(
+				srcPort, srcChannel, coins, sender, receiver, timeoutHeight, timeoutTimestamp,
+			)
 			if err := msg.ValidateBasic(); err != nil {
 				return err
 			}
 
 			return authclient.GenerateOrBroadcastMsgs(cliCtx, txBldr, []sdk.Msg{msg})
 		},
 	}
+	cmd.Flags().Uint64(flagTimeoutHeight, types.DefaultPacketTimeoutHeight, "Timeout height relative to the current block height. The timeout is disabled when set to 0.")
```

Comment:
I think doing relative timeouts in another pr is fine, but we should update both defaults to be 0 so the user is forced to enter some value